kmod/igb/igb_ptp.c

```diff
@@ -369,17 +365,14 @@ static int igb_ptp_gettime64_i210(struct ptp_clock_info *ptp,
 
 #ifdef HAVE_PTP_CLOCK_INFO_GETTIME64
 static int igb_ptp_settime64_82576(struct ptp_clock_info *ptp,
-				 const struct timespec *ts64)
+				   const struct timespec *ts64)
```

Comment:
Why the parameter type wasn't changed here as in other functions? Shouldn't we have `const struct timespec64 *ts64` here?